__tests__/lamp.test.js

```diff
@@ -16,20 +26,41 @@ describe('Lamp', () => {
 });
 
 describe('switchOn', () => {
-    test('it switches the light on', () => {
-        const lamp = new Lamp(jest.fn());
+    test('throws an error if bulb is dead', () => {
+        const lamp = new Lamp(bulb);
+        bulb.isWorking = false;
+        expect(() => {
+            lamp.switchOn()
+        }).toThrow('Bulb is dead');
+    });
+    test('It switches the light on', () => {
+        const lamp = new Lamp(bulb);
         lamp.switchOn();
         expect(lamp.isOn).toBe(true);
     });
+
+    describe('electrify', () => {
+        test('electrify gets called', () => {
+            const bulb = {
+            shelfLife: 5,
+            isWorking: true,
+            electrify: jest.fn()//test-doubles: stub
+            };
+            const lamp = new Lamp(bulb);
+            lamp.switchOn();
+            //test-doubles: spy
```

Comment:
As with previous comments. If this is actually a requirement to comment like this then fixing the format would be great